google-cloud-platform/app-engine/run.md

```diff
@@ -150,20 +154,10 @@ This step can take awhile but needs to be done only once.
 
 ## Run the app using App Engine
 
-Run the application using the following `gcloud preview app`
+Run the application using the following `dev_appserver.py --custom_entrypoint "dart bin/server.dart {port}" app.yaml`
```

Comment:
Please add a comment about that the CWD of the Dart VM process will be the directory where `app.yaml` is found, e.g. if doing:
```
dev_appserver.py --custom_entrypoint "dart bin/server.dart {port}" xxx/yyy/app.yaml
```
`bin/server.dart` should not be prefixed by `xxx/yyy`.